### DISCUSSION TITLE
When to support AppImage？

### DISCUSSION THREAD

**Discussion by 8MiYile** [obsproject/obs-studio#8712] [Development Questions] [answered]
When does GitHub Actions' build system support compiling appimage executables?

**Comment by Fenrirthviti** [comment, marked as answer]
The core team is not interested in putting time or effort in to providing AppImage packages at this time.

**Reply by 8MiYile** [reply]
Keep it for now, you can try it when you have time

**Reply by Fenrirthviti** [reply]
I would recommend reading the entirety of the comments on this PR, it's unlikely anyone from the core team will ever have an interest in working with anything AppImage, as the main developers of AppImage have been banned from our community.

https://github.com/obsproject/obs-studio/pull/2868

**Reply by 8MiYile** [reply]
Oh, it's a shame.
